draft-ietf-quic-http.md

```diff
@@ -1206,22 +1209,22 @@ function by communicating with clients.
 Servers initiate the shutdown of a connection by sending a GOAWAY frame
 ({{frame-goaway}}).  The GOAWAY frame indicates that client-initiated requests
 on lower stream IDs were or might be processed in this connection, while
-requests on the indicated stream ID and greater were not accepted. This enables
+requests on the indicated stream ID and greater were rejected. This enables
 client and server to agree on which requests were accepted prior to the
 connection shutdown.  This identifier MAY be lower than the stream limit
 identified by a QUIC MAX_STREAM_ID frame, and MAY be zero if no requests were
 processed.  Servers SHOULD NOT increase the QUIC MAX_STREAM_ID limit after
 sending a GOAWAY frame.
 
-Once GOAWAY is sent, the server MUST cancel requests sent on streams with an
+Once GOAWAY is sent, the server MUST reject requests sent on streams with an
 identifier higher than the indicated last Stream ID.  Clients MUST NOT send new
 requests on the connection after receiving GOAWAY, although requests might
 already be in transit. A new connection can be established for new requests.
 
 If the client has sent requests on streams with a higher Stream ID than
-indicated in the GOAWAY frame, those requests are considered cancelled
+indicated in the GOAWAY frame, those requests are considered rejected
 ({{request-cancellation}}).  Clients SHOULD reset any streams above this ID with
-the error code HTTP_REQUEST_CANCELLED.  Servers MAY also cancel requests on
+the error code HTTP_REQUEST_REJECTED.  Servers MAY also reject requests on
```

Comment:
Why have the client send REJECTED in this case?

Reply:
Because GOAWAY already tells the client that the server didn't act on these higher streams; this is effectively the client's agreement that these streams were rejected by GOAWAY.  (There's probably also a STOP_SENDING en route with the same error code.)

Reply:
This seems weird.  I think I prefer the client sending CANCELLED.

Reply:
Oh, I missed that.  Yes.  The client CAN'T send _REJECTED.  Because if the server echoes it (as they are allowed to), then the client might conclude that nothing was processed.  We need a prohibition against clients using _REJECTED.

> Clients MUST NOT use HTTP_REQUEST_REJECTED with STOP_SENDING.

Reply:
It's not that the client can't, because those streams *haven't* been processed, per GOAWAY.  But yes, we should add a prohibition on its use in other cases at least.